package.json

```diff
@@ -19,15 +19,18 @@
   "author": "Safe (https://safe.global)",
   "license": "MIT",
   "devDependencies": {
+    "jest": "^29.4.3",
+    "ts-jest": "^29.0.5",
+    "@types/jest": "^29.4.0",
+    "ts-node": "^10.9.1",
     "@types/node": "^18.11.18",
+    "typescript": "^4.9.4",
```

Comment:
Did you move this manually? Typescript doesn't have the @ so it should be at the bottom